extensions/resteasy-reactive/rest/deployment/src/main/java/io/quarkus/resteasy/reactive/server/deployment/ResteasyReactiveProcessor.java

```diff
@@ -634,6 +635,9 @@ public Supplier<Boolean> apply(ClassInfo classInfo) {
                         }
                     });
 
+            allowNotRestParametersBuildItem
+                    .ifPresent(skip -> serverEndpointIndexerBuilder.skipAllNotMethodParameter(true));
```

Comment:
This is faster and avoids a lambda class
```suggestion
            serverEndpointIndexerBuilder.skipAllNotMethodParameter (allowNotRestParametersBuildItem.orElse(false));
```